docs/releases/1.11.1.txt

```diff
@@ -78,3 +78,6 @@ Bugfixes
 
 * Prevented migrations from dropping database indexes from ``Meta.indexes``
   when changing ``Field.db_index`` to ``False`` (:ticket:`28052`).
+
+* Fixed an issue with ``ArrayField(CIText)`` values when retrieved from the
```

Comment:
Better? "Corrected the return type of ArrayField(CITextField()) values retrieved from the database"